src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const bigNumOne = new Big(numberOne);
+  const bigNumTwo = new Big(numberTwo);
+  let result;
+
+  if (operation === '-') {
+    result = bigNumOne - bigNumTwo;
```

Comment:
- Use `bigNumOne.minus(bigNumTwo)`. This is a function that already exists and it makes your code more readable.

Reply:
Implemented